reference/carts.v3.yml

```diff
@@ -3072,9 +3072,21 @@ components:
               cart_url:
                 type: string
                 format: url
+                example: 'https://store.mybigcommerce.com/cart.php?action=load&id={{id}}&token={{token}}'
               checkout_url:
                 type: string
                 format: url
+                example: 'https://store.mybigcommerce.com/cart.php?action=loadInCheckout&id={{id}}&token={{token}}'
+              embedded_checkout_url:
+                type: string
+                format: url
+                example: 'https://store.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id={{id}}&token={{token}}'
+          examples:
+            example-1:
+              value:
+                cart_url: 'https://store.mybigcommerce.com/cart.php?action=load&id={{id}}&token={{token}}'
```

Comment:
in these examples, it seems right not to put a fake id and token. However, it would be helpful to clarify the data type of the id and what entity it's the ID of.